.github/workflows/conda.yml

```diff
@@ -35,6 +38,12 @@ jobs:
       - uses: actions/checkout@v3
         with:
           fetch-depth: 0
+      - name: Setup Rust Toolchain
+        uses: actions-rs/toolchain@v1
+        id: rust-toolchain
+        with:
+          toolchain: stable
+          override: true
```

Comment:
Do we still need this? From what I gather looks like builds started passing once we removed the Substrait components, is that correct? I would think this wouldn't have an impact on whether or not the conda build succeeds based on our previous attempts